about.html

```diff
@@ -1,42 +1,43 @@
-<!DOCTYPE html>
-<html lang="en">
+<!doctype <!DOCTYPE html>
+<html>
```

Comment:
shouldn't we keep the lang?

Reply:
that tag I have found doesn't matter too much.  We will be cleaning the rest of the items up in class today.  As well as adding the necessary items to complete it.